db/app.js

```diff
@@ -4,6 +4,10 @@ const { serverError } = require("./models/error-handlers");
 
 const app = express();
 
+app.get("/api/", (req, res, next) => {
```

Comment:
we can separate this into a controller but you're right that we don't need a model!